src/docs/cookbook/testing/widget/introduction.md

```diff
@@ -154,8 +154,8 @@ which is particularly useful while testing.
 
 With a widget in the test environment, search
 through the widget tree for the `title` and `message`
-Text widgets using a `Finder`. This allow verification that
-the widgets are displaying correctly.
+Text widgets using a `Finder`. This allows verification that
+the widgets are being displayed correctly.
```

Comment:
I would have made it  “are correctly displayed.”   ;)

But it’s fine.

Reply:
Aw, yours is better. I could've changed it. Thanks ;)